map/map_projection_loader/package.xml

```diff
@@ -18,6 +18,7 @@
   <depend>rclcpp</depend>
   <depend>rclcpp_components</depend>
   <depend>tier4_map_msgs</depend>
+  <depend>yaml-cpp</depend>
```

Comment:
Note that it's already included in CMakeLists.txt. And as long as it's declared in CMakeLists.txt, I would prefer not to add it here.

I didn't include it in package.xml becaulse deleting this from CMakeLists.txt results in build failure (link error to yaml-cpp). Ideally we should fix this issue so that we don't have to declare yaml-cpp independently in CMakeLists.txt, but haven't figured out the solution yet.

But I'd like to hear your thoughts as well.